src/Runner.jl

```diff
@@ -1,6 +1,11 @@
 import Base: strip
 abstract type Runner; end
 
+# Host platform _must_ match the C++ string ABI of the binaries we get from the
+# repositories.  Note: when preferred_gcc_version=v"4" we can't really build for
+# that C++ string ABI :-(
```

Comment:
```suggestion
# Host platform _must_ match the C++ string ABI of the binaries we get from the
# Alpine repositories, so that when using `$HOSTCC` to compile host executables,
# they can link against e.g. `libboost` provided through `apk`.
# Note: when preferred_gcc_version=v"4" we can't really build for that C++ string
# ABI, so GCC 5 will be chosen instead.
```

Reply:
> so GCC 5 will be chosen instead

That's not true unfortunately :disappointed: